sdk/mx.sdk/mx_sdk_vm.py

```diff
@@ -649,19 +661,22 @@ def jlink_new_jdk(jdk, dst_jdk_dir, module_dists,
         jlink.append('--keep-packaged-modules=' + join(dst_jdk_dir, 'jmods'))
 
         if jdk_has_new_jlink_options(jdk):
-            if jdk_omits_warning_for_jlink_set_ThreadPriorityPolicy(jdk):
-                thread_priority_policy_option = ' -XX:ThreadPriorityPolicy=1'
-            else:
-                mx.logv('[Creating JDK without -XX:ThreadPriorityPolicy=1]')
-                thread_priority_policy_option = ''
-
-            if any((m.name == 'jdk.internal.vm.compiler' for m in modules)):
-                jlink.append('--add-options=-XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCIProduct -XX:-UnlockExperimentalVMOptions' + thread_priority_policy_option)
+            if jdk_supports_enablejvmciproduct(jdk):
```

Comment:
I don't think `-XX:ThreadPriorityPolicy` is related to -XX:+EnableJVMCIProduct is it?

Reply:
No, but isn't `-XX:ThreadPriorityPolicy=1` intended for libgraal usage? If so, isn't `EnableJVMCIProduct` [enabling libgraal](https://bugs.openjdk.java.net/browse/JDK-8232118). That was my thinking anyway. Happy to change that, though.

Reply:
No, setting `ThreadPriorityPolicy` is independent of libgraal.

Reply:
OK, thanks! Should be fixed now.